ocpp/v201/enums.py

```diff
@@ -1330,7 +1330,10 @@ class StatusInfoReasonType(str, Enum):
 
 
 class SecurityEventType(str, Enum):
-    """Security Events as listed in Appendices (Appendix 1. Security Events)"""
+    """
+    Security Events as listed in Appendices (Appendix 1. Security Events)
+    Updated to include v1.3
```

Comment:
This line looks more like a commit message. Does it belong here?

Reply:
I've reduced it down to just state v.13